tests/system/test_shutdown_message/test_shutdown_message.py

```diff
@@ -91,9 +98,14 @@ def test_shut_down_message_gracefully_stopped_agent(restart_all_agents, stop_gra
         expected_output:
             - Gracefully closed, it is expected to find agents 'Disconected' in agent-manager
     '''
-    time.sleep(TIMEOUT_AGENT)
-
-    matches = re.findall(r"Disconnected", host_manager.run_command(testinfra_hosts[0],
+    time.sleep(T_10)
+    shutdown_messages = 0
+
+    HostMonitor(inventory_path=inventory_path, messages_path=messages_path, tmp_path=tmp_path).run()
+    ossec_log = host_manager.get_file_content(test_infra_managers[0], '/var/ossec/logs/ossec.log')
```

Comment:
Hardcoded log file. Consider using a variable instead

Reply:
Done in b84dfe86234c0d1a3a97f